test/v1/server/document_test.go

```diff
@@ -45,13 +45,17 @@ type DocumentSuite struct {
 	database   string
 }
 
-func expect(s httpexpect.LoggerReporter) *httpexpect.Expect {
+func expectLow(s httpexpect.LoggerReporter, url string) *httpexpect.Expect {
```

Comment:
Wonder if a generic `GetBaseURL()` pattern might eliminate the need of having to implement this function.

Reply:
Sound like a good idea, I'll fix it in one of the next iterations.